effective-image-guide/readme.md

```diff
@@ -0,0 +1,56 @@
+>**The goal of this guide:** to explore what makes an image effective in relation to an article.
+
+# Sciencing the $@%* out of images
+
+When used correctly, effective visuals are extremely powerful to communicate an appropriate message. Let's look at some of the science behind visual content and what makes an image effective.
+
+## First off: Why visual content is so important
+
+Here are some telling stats as researched by [Buffer's](https://buffer.com/)  [Kevan Lee](https://blog.bufferapp.com/author/kevanlee) on just how big visual content has become and how it’s changing the way we all approach marketing:
+
+**63% of social media is made up of images.** That means nearly two-thirds of the updates you see on social media are visual content, according to a [Citrix report](http://www.slideshare.net/citrix/citrix-mobileusage) from 2014.
```

Comment:
I would like to know how they came up with this number. Did little time research, it looks solid, but don't know for example how big their research group was. Also, is this about the world population, Americans, Western world? Just when we want to science the &%$# out of things we should be critical of our sources.

But, yeah **maybe another achievable**.